src/core/opamDirTrack.mli

```diff
@@ -41,6 +41,11 @@ val track:
   OpamFilename.Dir.t -> ?except:OpamFilename.Base.Set.t ->
   (unit -> 'a OpamProcess.job) -> ('a * t) OpamProcess.job
 
+val track_t:
+  [< `Top of OpamFilename.Dir.t | `Specific of OpamFilename.Dir.t * string list ] ->
```

Comment:
It's generally considered better practice for interfaces such as this one to just export two functions rather than complexify the arguments (even if they will both call into the same code, of course).